README.md

```diff
@@ -197,7 +175,10 @@ Default: `null`
 
 ### <a name="input_managed_identities"></a> [managed\_identities](#input\_managed\_identities)
 
-Description: Managed identities to be created for the resource.
+Description:   Controls the Managed Identity configuration on this resource. The following properties can be specified:
+
+  - `system_assigned` - (Optional) Specifies if the System Assigned Managed Identity should be enabled.
```

Comment:
i have to keep this here or the linting fails, i think we should raise an issue with the AVM teams since we are not using system_assigned at all 
```suggestion
  - `system_assigned` - (Optional) Specifies if the System Assigned Managed Identity should be enabled.
```